README.md

```diff
@@ -35,10 +53,9 @@ Give a ⭐ if you like this project!
 
 ## Acknowledgments
 
-    Hat tip to anyone whose code was used 🔰
-    Microverse program ⚡
-    My standup team 🏹
-    My family's support 🙌
+   - Microverse learning partner
+   - Morning session team
+   - Microverse
 
 
 ## 📝 License
```

Comment:
[OPTIONAL]
- Provide a link to your License
- Add your ``.vscode``  directory into your ``.gitignore``.